drafts/25-xxx-specifications.txt

```diff
@@ -686,6 +690,9 @@ dfc50. When determining argument boundaries in the invocation of a
        sets of `[  ]` or `(/  /)` brackets, in addition to matching
        sets of `(  )` parentheses.
 
+dfc60. The token-list in the FPP `#pragma` directive is always
+       subject to macro replacement, and may not be empty.
```

Comment:
Are we sure we want this? It is a divergence from CPP. If so it should be documented in the appendix.

Reply:
```suggestion
dfc60. The token-list in the FPP `#pragma` directive may not be empty.
```